Project.toml

```
@@ -17,8 +17,8 @@ Scratch = "6c6a2e73-6563-6170-7368-637461726353"
 [compat]
 ColorTypes = "0.9, 0.10, 0.11, 0.12"
 Downloads = "1.3"
-FFMPEG = "0.3, 0.4"
-FFMPEG_jll = "4.1"
+FFMPEG = "0.3, 0.4, 1"
```

```suggestion
FFMPEG = "0.4.3"
```
https://github.com/JuliaIO/FFMPEG.jl/pull/59/files#r1896927588